api/v1/routes/blog.py

```diff
@@ -299,3 +307,23 @@ async def update_blog_comment(
         status_code=200,
         data=jsonable_encoder(updated_blog_comment)
     )
+
+
+@blog.delete("/likes/{blog_like_id}", 
+             status_code=status.HTTP_204_NO_CONTENT)
+async def delete_blog_like(
+    blog_like_id: str,
+    db: Session = Depends(get_db),
+    current_user: User = Depends(user_service.get_current_user),
+):
+    """Endpoint to delete `BlogLike`
+
+    args:
+        blog_like_id: `str` The ID of the BlogLike object.
+        request: `default` Request.
+        db: `default` Session.
+    """
+    blog_like_service = BlogLikeService(db)
+
+    # delete blog like
+    return blog_like_service.delete(blog_like_id, current_user.id)
```

Comment:
Kindly Add an acknowledgment to return to the consumer after blog dislike-delete.

From your Blog_dislike_service and the screenshot, None is been returned after a dislike is deleted.

Reply:
Boss, thank you for the review.

I have no issue making changes. But I'm aware of a sort of a recent consensus to not return anything but just pass `response_code=204`  for delete endpoints. @joboy-dev requested changes on [This PR](https://github.com/hngprojects/aivideo_be/pull/373) yesterday because of that.

But I'll update it if you still want me to.